tests/openapi/test_parameters.py

```diff
@@ -15,7 +15,7 @@ def test_create_parameters():
     route_handler = PersonController.get_persons
     parameters = create_parameters(
         route_handler=route_handler,
-        handler_fields=create_function_signature_model(fn=cast(Callable, route_handler.fn)).__fields__,
+        handler_fields=create_function_signature_model(fn=cast(Callable, route_handler.fn), plugins=[]).__fields__,
```

Comment:
do you have to send a blank list for plugins?

Reply:
its a required parameter, so an empty list it is.